react-hot-button-component/my-app/src/HotButton.js

```diff
@@ -0,0 +1,44 @@
+import './HotButton.css';
+import { useState } from 'react';
+
+export default function HotButton() {
+  const [counter, setCounter] = useState(0);
+
+  let color;
+  let textColor;
+
+  function handleClick() {
+    setCounter(counter + 1);
+  }
+
+  console.log(counter);
```

Comment:
Remove all console logs before submitting!

Reply:
Oh, thought I should leave it there so you could see what my counter was. Though I guess I should have shown the actual counter inside the app

Reply:
Ahh you're right, I didn't even catch that! Need to include it in your JSX